tests/postgres_tests/migrations/0002_create_test_models.py

```diff
@@ -182,6 +182,27 @@ class Migration(migrations.Migration):
             ]
         ),
         migrations.CreateModel(
+            name='Map',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=128)),
+            ],
+            options={
+                'indexes': [],
+            },
+        ),
+        migrations.CreateModel(
+            name='MapSpot',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('position', ArrayField(base_field=models.IntegerField(), size=2)),
+                ('map', models.ForeignKey(on_delete=models.CASCADE, to='postgres_tests.Map')),
+            ],
+            options={
+                'indexes': [],
```

Comment:
You need to add `'required_db_vendor': 'postgresql',` in `options` so the model with `ArrayField` isn't created on other databases.